web/src/compose_tooltips.js

```diff
@@ -30,7 +30,13 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: ".compose_control_button",
+        // This specificity is necessary in order to resolve the conflict
+        // between these two classes and ensure the tooltip is displayed correctly.
+        target: `
+        .compose_control_button[data-tooltip-template-id],
```

Comment:
I'm wondering if there's a way to simplify this...

Reply:
Not without disturbing other styles and things, I don't think.

I would update the comment to say something like "Only display Tippy content on classes accompanied by a `data-` attribute."